test/phpunit/dkan_harvest/DatajsonHarvestMigrationScenariosTest.php

```diff
@@ -495,7 +521,6 @@ public function testResourceRedirect() {
    * drops some taxonomies which will make following tests fail.
    */
   public function testHarvestError() {
-    return;
```

Comment:
This was a left over from a previous debugging session. Dropped the return statement to re-enable the test.